frontend/src/views/MapView.vue

```diff
@@ -3,63 +3,66 @@
     <v-progress-circular indeterminate :size="128"></v-progress-circular>
   </v-overlay>
 
-  <v-container v-if="!mdAndDown" fluid>
-    <v-row fill-height style="height: calc(100vh - 165px)">
+  <v-container fluid>
+    <div id="div-plot-button" class="desktop-plot-buttons-container">
+      <v-card
+        location="left"
+        variant="flat"
+        :style="{
+          backgroundColor: 'transparent'
+        }"
+        max-width="500"
+        max-height="145"
+      >
+        <v-btn
+          style="margin-right: 10px"
+          @click="toggleMetadata"
+          :color="showMetadata ? 'blue' : 'white'"
+        >
+          Metadata
+        </v-btn>
+        <v-btn
+          id="btn-show-historical"
+          style="margin-right: 10px"
+          @click="toggle('historical')"
+          :color="showHistorical ? 'blue' : 'white'"
+        >
+          Historical
+        </v-btn>
+        <v-btn
+          style="margin-right: 10px"
+          @click="toggle('forecast')"
+          :color="showForecast ? 'blue' : 'white'"
+        >
+          Forecast
+        </v-btn>
```

Comment:
it would be nice to have feedback to indicate that the data are loading. Right now it initially just loads a blank plot with no "loader" or "spinner" to indicate progress...